simple_ai_agents/Career_Guidence/career_guidance_system.py

```
@@ -0,0 +1,637 @@
+                    {"role": "user", "content": user_query}
+                ]
+            )
+            
+            answer = response.content
+            
+            # Ensure the response is properly formatted as HTML
+            if "<" not in answer:
+                # If no HTML tags are present, add basic formatting
+                answer = f"<div><p>{answer}</p></div>"
+            
+            return answer
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)